tests/CMakeLists.txt

```diff
@@ -143,6 +143,7 @@ if (UNIX
     if (BUILD_OPENSSL)
       add_subdirectory(openssl)
       add_subdirectory(openssl_unsupported)
+      add_subdirectory(openssl_moretest)
```

Comment:
Please choose a descriptive name for the set of openssl tests being added e.g. `openssl_digest` or `openssl_sgx_tcrypto_compat`.

Reply:
`openssl_sgx_tcrypto_compat` is OK. There will be more test cases than digest test in the same folder later. Change has been committed.